Package.resolved

```diff
@@ -73,6 +91,24 @@
           "version": "1.0.1"
         }
       },
+      {
+        "package": "SQL",
+        "repositoryURL": "https://github.com/vapor/sql.git",
+        "state": {
+          "branch": null,
+          "revision": "f77bcae7c95150cdab62095e7bf321ed8c7551bd",
+          "version": "2.3.0"
+        }
+      },
+      {
+        "package": "SQLite",
+        "repositoryURL": "https://github.com/vapor/sqlite.git",
+        "state": {
+          "branch": null,
+          "revision": "4f5e173abafb0bb185608d28fccec3df166c689a",
+          "version": "3.2.0"
+        }
+      },
```

Comment:
Why is this back?

Reply:
don't know 🤔 `swift package generate-xcodeproj` added it back

Reply:
Should be gone now with usage of `upToNextMinor`, but don't know what's going on there